### DISCUSSION TITLE
hello i need your help please

### DISCUSSION THREAD

**Discussion by SkTeamR** [CobanRamo/Lenovo-IdeaPad-S340-15IIL#14] [General]
I currently have a Lenovo IdeaPad S340-15IIL laptop, in which I need to put mac os, so that later on I can use it with dualboot. I have followed many tutorials but I can't get it to install.

Bios: CUCN28WW(V1.17)

**Comment by SkTeamR** [comment]
**this is how i have my usb drive**
<img src="https://i.ibb.co/WP745Xn/1.png" alt="1" border="0">

**that is the first error that shows me when doing the boot**
<img src="https://i.ibb.co/h8yDRt3/2.jpg" alt="2" border="0">

**it only comes out like this and does not allow the installation**
<img src="https://i.ibb.co/Fm625Yp/3.jpg" alt="3" border="0">

**Comment by CobanRamo** [comment]
Hi,

So, first of all, you have formatted the start medium incorrectly.
![Bildschirmfoto 2022-03-22 um 00 18 32](https://user-images.githubusercontent.com/68192365/159378971-4065a6db-919b-4173-8189-711830d6b9e3.png)

This must be a GPT style GUID partition table and formatted with FAT32.
![Bildschirmfoto 2022-03-22 um 00 23 06](https://user-images.githubusercontent.com/68192365/159379042-2827e06c-3356-4275-b4ed-b8c852dc5aab.png)

Also, the folder structure on this USB stick must look like the picture.
![Bildschirmfoto 2022-03-22 um 00 26 57](https://user-images.githubusercontent.com/68192365/159379359-5b74452b-1e89-44c2-96b8-10c9257e236e.png)

For a USB stick with the "com.apple.recovery.boot" recovery structure, your EFI must have a working network driver.
You have to remember, the recovery that starts like this can't do an installation, it loads the MacOS from the Apple server, so a working internet connection is mandatory.

Remember, you should prepare your built-in nvme/SSD/HDD for an installation, for a dualboot on the same media you must first of all have a min. 200MB EFI partition, for example after Windows partition etc. you must also create free space for MacOS installation which will be formatted in the recovery installer to apfs container.

**Comment by SkTeamR** [comment]
hello, thanks for your answer and if it seems like it was the error of the USB format since it was not as GPT, now I managed to start, but I'm stuck at this part 😞
<img src="https://i.ibb.co/rcV8zq2/16479603689078268713241562457371.jpg" alt="16479603689078268713241562457371" border="0">

**Comment by CobanRamo** [comment]
Is it possible that you have the wrong bios settings?
![Bildschirmfoto 2022-03-23 um 00 06 58](https://user-images.githubusercontent.com/68192365/159590893-1933037e-02d6-423c-8575-c7bd11b5a5e9.png)

![Bildschirmfoto 2022-03-23 um 00 04 10](https://user-images.githubusercontent.com/68192365/159590921-cabaa59c-4c6d-4a8f-a631-6b5940fec9e5.png)

![Bildschirmfoto 2022-03-23 um 00 05 02](https://user-images.githubusercontent.com/68192365/159591289-efc65cf8-734e-4432-b72a-1d2fa5d291f5.png)
